ietf/doc/migrations/0050_editorial_stream_states.py

```diff
@@ -0,0 +1,43 @@
+# Copyright The IETF Trust 2022, All Rights Reserved
+from django.db import migrations
+
+
+def forward(apps, schema_editor):
+    State = apps.get_model("doc", "State")
+    StateType = apps.get_model("doc", "StateType")
+    StateType.objects.create(
+        slug="draft-stream-editorial", label="Editorial stream state"
+    )
+    for slug, name, order in (
+        ("repl", "Replaced editorial stream document", 0),
+        ("active", "Active editorial stream document", 2),
```

Comment:
order is almost a nuisance field. I think we'll have less confusion with this migration as is. We can make order look prettier (to no good effect otherwise) later.